### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by mouuii**

### CODE REVIEW

File: internal/net/call/call_test.go [hunk 1]
```diff
@@ -29,12 +29,12 @@ import (
 	"testing"
 	"time"
 
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
-	"go.opentelemetry.io/otel/trace"
 	"github.com/ServiceWeaver/weaver/internal/cond"
 	"github.com/ServiceWeaver/weaver/internal/net/call"
 	"github.com/ServiceWeaver/weaver/internal/traceio"
 	"github.com/ServiceWeaver/weaver/runtime/logging"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
```
Comment (rgrandl) [inline internal/net/call/call_test.go:36]: Can you don't change the order of these imports?

**Review by rgrandl — COMMENTED** [review]
Thanks for fixing the typos. I left a small comment.

File: internal/net/call/call_test.go (hunk 1 above)
Comment (spetrovic77) [reply internal/net/call/call_test.go:36]: I've seen this happen to me too, I think different IDEs have different ideas maybe?

Comment (spetrovic77) [reply internal/net/call/call_test.go:36]: Maybe okay to let it happen in a few files, as long it's not the entire repo :).

**Review by spetrovic77 — APPROVED** [review]
Wath typos? ;)

Thanks for fixing!

File: internal/net/call/call_test.go (hunk 1 above)
Comment (mouuii) [reply internal/net/call/call_test.go:36]: thank you, this is because of the goland IDE , I have turned it off

**Pull request merged**